.github/workflows/ci.yml

```diff
@@ -26,6 +26,10 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Shellcheck
         run: shellcheck pantsup.sh
+      - uses: luizm/action-sh-checker@v0.5.0
```

Comment:
This runs on every `.sh` file?

Reply:
yes. will add some PR to address the issues detected by this check.
see: https://github.com/pantsbuild/setup/pull/139 (first one), will have more.
so this PR can pass CI checks.